codegen/src/test/java/software/amazon/awssdk/codegen/poet/auth/scheme/AuthSchemeSpecTest.java

```diff
@@ -161,11 +161,24 @@ static List<TestCase> parameters() {
                     .caseName("ops-with-no-auth")
                     .outputFileSuffix("default-provider")
                     .build(),
-            // Service with signature version with the same value as S3
+            // S3
             TestCase.builder()
-                    .modelProvider(ClientTestModels::serviceMiniS3)
+                    .modelProvider(ClientTestModels::serviceS3)
+                    .classSpecProvider(EndpointBasedAuthSchemeProviderSpec::new)
+                    .caseName("s3-test")
+                    .outputFileSuffix("default-provider")
+                    .build(),
+            TestCase.builder()
+                    .modelProvider(ClientTestModels::serviceS3)
                     .classSpecProvider(ModelBasedAuthSchemeProviderSpec::new)
-                    .caseName("mini-s3")
+                    .caseName("s3-test")
```

Comment:
I changed it to use real s3 model because why not 🤷🏼‍♀️ and it may help us catch issues